src/Hazelcast.Net/Clustering/ClusterEvents.cs

```diff
@@ -623,21 +696,28 @@ private async ValueTask HandleCodecPartitionViewEvent(int version, IList<KeyValu
             // On... does not throw
             await _partitionsUpdated.AwaitEach().CfAwait();
         }
-        
-        private  ValueTask HandleCodecClusterVersionEvent(ClusterVersion version, object state)
+
+        private ValueTask HandleCodecClusterVersionEvent(ClusterVersion version, object state)
         {
-            _logger.LogDebug("Handle ClusterVersion event.");
+            _logger.IfDebug()?.LogDebug("Handle ClusterVersion event");
```

Comment:
Applies to several places in this file, but I'd like to see more information provided in these debugging logs - their primary purpose would be to provide information about a production issue, and the current messages provide very little to work with

Reply:
Sure, fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/902/commits/b246b6c0752b52869313042c36feb40935239ed6